test/CMakeLists.txt

```diff
@@ -2244,17 +2244,17 @@ ecbuild_add_test( TARGET  test_${PROJECT_NAME}_ghcn_snod
                           -m maskout"
                           ghcn_snod_20200228.nc ${IODA_CONV_COMP_TOL})
 
-ecbuild_add_test( TARGET  test_${PROJECT_NAME}_smap_ssm
+ecbuild_add_test( TARGET  test_${PROJECT_NAME}_smap_nrt_ssm
                     TYPE    SCRIPT
                     ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
                     COMMAND bash
                     ARGS    ${CMAKE_BINARY_DIR}/bin/iodaconv_comp.sh
                             netcdf
                             "${Python3_EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/smap_ssm2ioda.py
                             -i testinput/SMAP_L2_SM_P_NRT_24342_A_20190822T224858_N16023_002.h5
-                            -o testrun/smap_ssm.nc
+                            -o testrun/smap_nrt_ssm.nc
```

Comment:
@BenjaminRuston This file was not found in the test. Perhaps remove the test until we can get data?

Reply:
the `testrun` is the ctest output... here the issue was with the `testoutput` and needed a change in the CMakeLists.txt